modules/quanthub_datasetexplorer/src/Controller/DatasetExplorer.php

```diff
@@ -45,80 +45,6 @@ public static function create(ContainerInterface $container) {
     );
   }
 
-  /**
-   * Build Slices block.
-   */
-  public function slices(Request $request) {
```

Comment:
Please ask Ilya about should we delete it from the profile.

Reply:
let's remove /slices